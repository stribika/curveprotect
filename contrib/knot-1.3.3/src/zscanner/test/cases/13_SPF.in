$ORIGIN	.
$TTL	1

; The SPF is the same as the TXT, so there are the differences and basics only.

; OK
@	SPF	"" "test1" "\255" test2		; Array of text strings
@	SPF	\# 1 00				; Hexadecimal rdata
@	TYPE99	\# 1 00				; TYPE + Hexadecimal rdata
@	TYPE99	""				; TYPE
@	spf	""				; Type in lower-case

; KO
@	SPF
