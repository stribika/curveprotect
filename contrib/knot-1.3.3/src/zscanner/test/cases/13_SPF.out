OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0063
RDATA=0005746573743101FF057465737432
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0063
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0063
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0063
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0063
RDATA=00
------
WARNG=ZSCANNER_EBAD_RDATA
------
